lib/code_corps_web/controllers/category_controller.ex

```diff
@@ -1,17 +1,43 @@
 defmodule CodeCorpsWeb.CategoryController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  alias CodeCorps.Category
+  alias CodeCorps.{ Category, User, Helpers.Query}
 
-  plug :load_resource, model: Category, only: [:show]
-  plug :load_and_authorize_resource, model: Category, only: [:create, :update]
-  plug JaResource
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
 
-  @spec model :: module
-  def model, do: CodeCorps.Category
+  @spec index(Conn.t, map) :: Conn.t
+  def index(%Conn{} = conn, %{} = params) do
+    with category <- Category |> Query.id_filter(params) |> Repo.all do
```

Comment:
`categories` instead of `category` here. We don't want to imply that index loads a single record. I'm guessing a copy-paste lapse :)